### PR TITLE
Pre-release ignoring

### LINKED ISSUE #195: Ignore pre-release client releases

### PR DESCRIPTION
**Pull request by 1-alex98**
Fixes #195

### CODE REVIEW

**Comment by 1-alex98** [comment]
Currently deployed on test.faforever.com

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "json2csv": "3.7.0",
     "moment": "2.24.0",
     "moment-timezone": "0.5.10",
+    "node-schedule": "^1.3.2",
```
Comment (janmanfai) [inline package.json]: I'd appreciate it if you could provide the corresponding yarn.lock file.

File: express.js [hunk 2]
```diff
@@ -238,23 +239,25 @@ let getLatestClientRelease = require("./scripts/getLatestClientRelease");
 extractor.run();
 getLatestClientRelease.run();
 
-// Run leaderboard extractor every minute
-setTimeout(() => {
+// Run leaderboard extractor every 15 minutes
+schedule.scheduleJob('*/15 * * * *', function myFunc() {
```
Comment (janmanfai) [inline express.js]: An anonymous function is adequate in this case. The same applies to the other scheduleJob.

Comment (1-alex98) [reply express.js]: u mean an arrow function?
I am sorry I got not much of an idea if it comes to javascript?

File: package.json (hunk 1 above)
Comment (1-alex98) [reply package.json]: hmm, i got massive issue getting this website to run on windows. When I fix some issue the next one pops up...

Comment (1-alex98) [reply package.json]: Maybe I should install me Linux again I remeber it was easy there

File: express.js (hunk 2 above)
Comment (janmanfai) [reply express.js]: No worries. Just use function() so it looks like this `schedule.scheduleJob('*/15 * * * *', function() {`.

Comment (1-alex98) [reply express.js]: oh ok

**Comment by 1-alex98** [comment]
@janteine went for linux now

**Comment by Brutus5000** [comment]
@axel1200 Please resolve the conflict

File: express.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ let expressValidator = require('express-validator');
 let bodyParser = require('body-parser');
 let passport = require('passport');
 let OAuth2Strategy = require('passport-oauth2');
+let schedule = require('node-schedule');
```
Comment (louvenarde) [inline express.js]: Please do not add new dependencies to the website unless you need a new feature. The schedule job change you've added does exactly the same as an interval, except we now have more dependencies.

**Review by louvenarde — CHANGES_REQUESTED** [review]
The update of GetLatestClientRelease.js and middleware.js is good and needed
However I do not see the point of the update of express.js . It is not needed as I see it, and you're adding extra dependencies. Please keep the part of your PR that is about ignoring the pre-release, and drop the part this is about using an extra dependency to do a simple setInterval.

**Comment by Rackover** [comment]
(apart from that the PR is very nice, i do not see much worth nitpicking)

**Comment by Rackover** [comment]
Tidied it up to keep only the release ignoring, merging in

**Review by louvenarde — APPROVED** [review]

**Pull request merged**